routing.rst

```diff
@@ -1453,6 +1453,117 @@ A possible solution is to change the parameter requirements to be more permissiv
     as the token and the format will be empty. This can be solved by replacing
     the ``.+`` requirement by ``[^.]+`` to allow any character except dots.
 
+.. _routing-alias:
+
+Aliasing
+--------
+
+.. versionadded:: 5.4
+
+    Support for route aliases was introduced in Symfony 5.4.
+
+You may sometimes want to have multiple ids for the same route. You can do so by
```

Comment:
IMO the word « ids » is not the best
On routing doc it is named « route name »
Just my 2cts :)

Reply:
Updated.